lang/gcc13/Portfile

```diff
@@ -170,8 +170,19 @@ if { ${subport} ne ${libcxxname} && ${os.platform} eq "darwin" && ${subport} ne
 
 if { ${os.platform} eq "darwin" } {
     # Patch generated from https://github.com/iains/gcc-13-branch
-    # git diff --no-prefix releases/gcc-13.1.0 gcc-13.1-darwin-r0
+    # git diff --no-prefix releases/gcc-13.3.0 gcc-13.3-darwin-r1
     patchfiles-append      patch-darwin-gcc-${version}.diff
+
+    # Additional patches from https://github.com/iains/gcc-13-branch, from
+    # branch gcc-13-3-darwin, after tag gcc-13.3-darwin-r0, including macOS 15
```

Comment:
Do you really mean gcc-13.3-darwin-r0 here ?

Reply:
No, r1. I was led astray by what had been on line 173.

Updated.

Reply:
Ok thanks , i guessed as much. I indeed tended not to always update the versions in those comments as i put them there just as a reminder how I made the patch, not intending them to be documenting the specific versions. But i agree not doing that was confusing to others…